transit_odp/organisation/csv/overall.py

```diff
@@ -89,6 +95,54 @@
     "string_lines",
 )
 
+DATACATALOGUE_ATTRIBUTE_FIELDS = (
+    "fares_metadata_id",
+    "xml_file_name",
+    "national_operator_code",
+    "line_name",
+)
+
+
+def get_fares_data():
```

Comment:
All of the preexisting code had type annotations

Reply:
@ITO-PaulHolland By type annotations, do you mean this (adding data types to the naming):
<img width="221" alt="image" src="https://user-images.githubusercontent.com/75623814/212896522-1479c0e9-e6b4-4ea7-96f5-367c3b5241d4.png">